packages/sdk/utils/bigint.ts

```diff
@@ -0,0 +1,13 @@
+export const minimizeBigInt = (value: Uint8Array): Uint8Array => {
+    const minValue = [0, ...value]
+    for (let i = 0; i < minValue.length; ++i) {
+        if (minValue[i] === 0) {
+            continue
+        }
+        if (minValue[i] > 0x7f) {
+            return new Uint8Array(minValue.slice(i - 1))
+        }
+        return new Uint8Array(minValue.slice(i))
+    }
+    return new Uint8Array([0])
+}
```

Comment:
nit: can you have your editor to add new line at end of file. 😄 personal pet peeve of mine.

Reply:
Lol fair enough. I pushed a new version with new lines :-).